e2e/tests/integrations/admin/edit-integration.test.ts

```diff
@@ -0,0 +1,33 @@
+import { expect, test } from "@playwright/test";
+import { fakerNL_BE as faker } from "@faker-js/faker";
+import {
+  createIntegration,
+} from "./create-integration.js";
+import { IntegrationTypes } from "../../types.js";
+
+test.use({ storageState: "playwright/.auth/admin.json" });
+
+test("As an admin I can edit an existing integration", async ({ page }) => {
+  const { page: integrationPage, name: integrationName } =
+    await createIntegration(page, IntegrationTypes.SEARCH_API);
+
+  await expect(
+    page.locator("h1").getByText(`Integration Details: ${integrationName}`)
+  ).toBeVisible();
+
+  const url = integrationPage.url();
+  const integrationId = url.split("/").pop();
+
+  await page.goto(`/admin/resources/integrations/${integrationId}`);
+
+  const newIntegrationName = faker.word.adjective();
+
+  await page.getByTestId('edit-resource').click();
```

Comment:
No option to improve this selector?

Reply:
There's not really a label which I can use.
That's probably the reason why laravel nova added the `data-testid`